lib/mixlib/install/product.rb

```diff
@@ -198,18 +185,6 @@ def lookup(key, version = :latest)
     config_file "/etc/opscode/chef-server.rb"
   end
 
-  product "chef-server-ha-provisioning" do
```

Comment:
Has this been deprecated?  I don't know anything about it, but I know @irvingpop added it recently.